### DISCUSSION TITLE
Will v3.9.2 be released?

### DISCUSSION THREAD

**Discussion by meganetaaan** [Moddable-OpenSource/moddable#1121] [Q&A] [answered]
I noticed that there is still no release of v3.9.2 on the [release page](https://github.com/Moddable-OpenSource/moddable/releases). Could you please create release notes for it?
Currently, when installing the latest Moddable release on xs-dev, it pulls information from this Release section, resulting in the installation of v3.9.1. This leads to a [build failure of ElevenLabsTTS](https://github.com/meganetaaan/stack-chan/pull/171#issuecomment-1560647302) in Stack-chan's development environment setup.

**Comment by phoddie** [comment, marked as answer]
It is great to see the rapid progress to integrate the new text-to-speech support into Stack-chan!

We can post an interim release so that your community can easily begin using it.

**Background on releases**

We post an official release of the Moddable SDK once a month. We have found that more frequently releases can be overwhelming, both for our team and the developers. Our target is for the release to happen the beginning of the month

 Preparing a release is a **big** project -- running tests , creating binaries for all development platforms, writing release notes, etc. In between official releases, we do post updated software, as we did here for the MP3 / text-to-speech features, but we don't create a release for them. The updates are usually tagged those so they are convenient to refer to. A recent example is [3.8.7](https://github.com/Moddable-OpenSource/moddable/releases/tag/3.8.7). In that case, we do not provide release notes, deferring that to the monthly release.

**Reply by phoddie** [reply]
The [3.9.2 release](https://github.com/Moddable-OpenSource/moddable/releases/tag/3.9.2) is now live.

**Comment by meganetaaan** [comment]
Thank you for the interim release! I understood the release cycle.
Stack-chan's build workflow now success to build ElevenLabsTTS!
https://github.com/meganetaaan/stack-chan/pull/171/checks

**Reply by phoddie** [reply]
Excellent! We look forward to the demo. 😉
